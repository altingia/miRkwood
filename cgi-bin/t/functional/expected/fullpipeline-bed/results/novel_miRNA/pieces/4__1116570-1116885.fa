>4__1116570-1116885
UUUAAGGGUUAAGUUGUAUACUAGUCACUUUUUAAAAUUUAAUUGAAAAAAUAGUCACUUUACUAUUUAUAAUUUGCAAAAUAGUCACUUUUCACUGUUUGAUUCGUACACUUAGUUUGUACACCACUUUUAGUGAUCAGAUGUAUAUAAAAAAUACACAGCGUACACUUUACAUUUGUACAAUAAAAGUGAUGUACAAUCCAAGUGUACGAAUCAAACAGUGAAAAGUGACUACUUUGCAAAUUAUGAAUAGUAAAGUGACUACUUUCCUAAUUAAGUUUGAAAAAGUGACUAUUUUGCCAAAACACCCCUUGAA
