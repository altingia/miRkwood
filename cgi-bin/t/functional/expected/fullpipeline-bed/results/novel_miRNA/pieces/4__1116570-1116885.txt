>4__1116570-1116885, stemloop structure
UUUAAGGGUUAAGUUGUAUACUAGUCACUUUUUAAAAUUUAAUUGAAAAAAUAGUCACUUUACUAUUUAUAAUUUGCAAAAUAGUCACUUUUCACUGUUUGAUUCGUACACUUAGUUUGUACACCACUUUUAGUGAUCAGAUGUAUAUAAAAAAUACACAGCGUACACUUUACAUUUGUACAAUAAAAGUGAUGUACAAUCCAAGUGUACGAAUCAAACAGUGAAAAGUGACUACUUUGCAAAUUAUGAAUAGUAAAGUGACUACUUUCCUAAUUAAGUUUGAAAAAGUGACUAUUUUGCCAAAACACCCCUUGAA
((((((((.....(((((...((((((((((((.(((((((((((..(((.(((((((((((((((((((((((((((((.((((((((((((((((((((((((((((((((.(.(((((((.((((((((.((..((((((((.........................))))))))..)).)))))))).))))))).).)))))))))))))))))))))))))))))))).))))))))))))))))))))))))))))).)))..))))))))))).))))))))))))...)).))).....))))))))
